### ISSUE TITLE
TestSuite containing TestSuites

### ISSUE THREAD

**Issue by tkonieczny** [kubeshop/testkube#3933]
**Describe the enhancement you'd like to see**
Currently TestSuite can only contain Tests. It would be useful to also allow setting other TestSuites as part of the TestSuite. It would allow to easily aggregate purpose-specific TestSuites into bigger ones - easier testsuites maintenance, ability to run multiple TestSuites easily, etc.
For example:
![Screenshot from 2023-05-31 13-38-42](https://github.com/kubeshop/testkube/assets/1473096/ff779803-14a3-4d22-a97b-284eb4bbeaac)

**Comment by vsukhin** [comment]
yeah, we dicussed this idea with users from time to time. worth for @aabedraba and @TheBrunoLopes to think about. kinda need to avoid recursion or just allow one level depth

**Comment by exu** [comment]
Should we allow for recursion here? It's quite complex as test-suite-1 can contain test-suite-2 which can contain test-suite-1. 

If we are ok with move responsibility for managing cycles to the user it should be quite easy task.

**Comment by aabedraba** [comment]
@exu good point. 

I think we should warn users in the docs but eventually leave them do it. 

@tkonieczny happy to tackle this in 1.13
